app/models/admin_only/member_app_waiting_reason.rb

```diff
@@ -0,0 +1,25 @@
+module AdminOnly
+  
+  class MemberAppWaitingReason < ApplicationRecord
+
+    validates_presence_of :name_sv
+
+    def other_reason_placeholder?
+      name_sv == self.class.other_reason_name
+    end
+
```

Comment:
I'm not totally sure what you're doing here Ashley. Could you explain this one to me?